### ISSUE TITLE
Wide space between 2nd and 3rd toes

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11668]
Wide space between 2nd and 3rd toes

## Proposed definition

A widely spaced gap between the second toe and the third toe.

## Parent term

- `HP:0008094 Widely spaced toes`

## Synonyms

- Diastasis between the second and third toes
- Increased 2nd-3rd toe gap

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

---

**Comment by pnrobinson** [comment]
@evanc1648
Adding new term: 
- ID: HP:6001522
- label: Wide space between 2nd and 3rd toes
- parents: [Widely spaced toes (HP:0008094)](https://hpo.jax.org/app/browse/term/HP:0008094)
- pmid: PMID:19125433
- def: The width of the gap between the second and third toe is increased. This term should be used to describe a gap with normal width of the toes themselves (rather than to describe a situation where the toes are thin or narrow).
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11668
- synonyms: Diastasis between the second and third toes(NONE)|Increased 2nd-3rd toe gap(NONE)
